### PR TITLE
chore(eslint): align lint with prettier rules

### PR DESCRIPTION
**Pull request by frassinier**
ESlint did not match Prettier config regarding arrow-parens
Align ESlint config regarding arrow-parens to fit Prettier one

### Futher read

* https://eslint.org/docs/rules/arrow-parens
* https://eslint.org/docs/rules/no-mixed-operators

### CODE REVIEW

File: tools-javascript/.eslintrc [hunk 1]
```diff
@@ -2,9 +2,11 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
+    "arrow-parens": [2, "as-needed"],
     "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}],
-    "no-tabs": "off",
     "indent": ["error", "tab", { "SwitchCase": 1 }],
+    "no-mixed-operators": "off",
```
Comment (jsomsanith) [inline tools-javascript/.eslintrc]: Why did you add this one ?

Comment (frassinier) [reply tools-javascript/.eslintrc]: Because of Prettier which removes parenthesis
Take a look [here](https://github.com/Talend/ui/pull/829#pullrequestreview-74363107)

Comment (jsomsanith) [reply tools-javascript/.eslintrc]: shit, but ok
